scte35/state_test.go

```diff
@@ -705,3 +738,207 @@ func TestOutInInOutIn36_37_10_37_37(t *testing.T) {
 		t.Errorf("There should have been 1 open signal, but we saw (%d) open signals instead.", len(state.Open()))
 	}
 }
+
+// Tests closing logic of 0x11 with 0x10, 0x20 ,0x30
+func Test11ClosingLogic(t *testing.T) {
+	state := NewState()
+
+	// 0x10
+	scteDesc10SignalBytes, _ := base64.StdEncoding.DecodeString(scteDesc10Signal)
+	outSignal, err := NewSCTE35(append([]byte{0x0}, scteDesc10SignalBytes	...))
+	if err != nil {
+		t.Fatal("Error creating SCTE-35 signal, return err:", err.Error())
+	}
+
+	closed, err := state.ProcessDescriptor(outSignal.Descriptors()[0])
+	if err != nil {
+		t.Errorf("ProcessDescriptor returned an error: %s", err.Error())
```

Comment:
thoughts on this wording?
```suggestion
		t.Errorf("ProcessDescriptor returned an error unexpectedly: %s", err.Error())
```